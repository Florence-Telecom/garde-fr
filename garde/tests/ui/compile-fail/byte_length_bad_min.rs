#[derive(garde::Validate)]
struct Test<'a> {
    #[garde(byte_length(min = 100, max = 10))]
    field: &'a str,
}

fn main() {}
