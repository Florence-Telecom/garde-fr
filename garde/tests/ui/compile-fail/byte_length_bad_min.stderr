error: `min` must be lower than or equal to `max`
 --> tests/ui/compile-fail/byte_length_bad_min.rs:3:25
  |
3 |     #[garde(byte_length(min = 100, max = 10))]
  |                         ^^^
